### PR TITLE
[16.0][MIG] stock_picking_send_by_mail

### PR DESCRIPTION
**Pull request by moaadbourhim**
Migration from 15.0

### CODE REVIEW

**Comment by willmakenoise** [comment]
@rousseldenis It seems this should be good to merge. The tests aren't visible anymore, so can't see why it failed.

**Comment by willmakenoise** [comment]
@moaadbourhim Can you update the test? The field move_lines on stock.picking is renamed to move_line_ids. Should be a quick fix.

**Review by jorgecurso1 — APPROVED** [review]

File: stock_picking_send_by_mail/tests/test_stock_picking_send_by_mail.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        cls.product = cls.env["product.product"].create({"name": "Test product"})
+        cls.location_id = cls.env.ref("stock.stock_location_stock")
+        cls.location_destination_id = cls.env.ref("stock.stock_location_customers")
+        cls.picking_type = cls.env.ref("stock.picking_type_out")
+        cls.picking = cls.env["stock.picking"].create(
+            {
+                "picking_type_id": cls.picking_type.id,
+                "location_id": cls.location_id.id,
+                "location_dest_id": cls.location_destination_id.id,
+                "move_lines": [
```
Comment (rousseldenis) [inline stock_picking_send_by_mail/tests/test_stock_picking_send_by_mail.py:20]: @moaadbourhim You need to change this as the filed does not exist anymore

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by legalsylvain** [comment]
hi @moaadbourhim. Could you take a look on @rousseldenis remarks ? Thanks !

**Comment by pedrobaeza** [comment]
Superseded by #1474

**Pull request closed without merging**